packages/cspell-types/src/CSpellSettingsDef.ts

```diff
@@ -119,18 +131,49 @@ export interface FileSettings extends ExtendableSettings, CommandLineSettings {
     features?: Features;
 }
 
+/* eslint-disable no-irregular-whitespace */
+/**
+ * In the below JSDoc comment, we helpfully specify an example configuration for the end-user to
+ * reference. And this example will get captured by the automatic documentation generator.
+ *
+ * However, specifying the glob pattern inside of a JSDoc is tricky, because the glob contains the
+ * same symbol as the end-of-JSDoc symbol. To work around this, we insert a zero-width space in
+ * between the "*" and the "/" symbols.
+ */
 export interface ExtendableSettings extends Settings {
-    /** Overrides to apply based upon the file path. */
+    /**
+     * Overrides are used to apply settings for specific files in your project.
+     *
+     * For example:
+     *
+     * ```javascript
+     * "overrides": [
+     *   // Force `*.hrr` and `*.crr` files to be treated as `cpp` files:
+     *   {
+     *     "filename": "**​/{*.hrr,*.crr}",
+     *     "languageId": "cpp"
+     *   },
+     *   // Force `*.txt` to use the Dutch dictionary (Dutch dictionary needs to be installed separately):
+     *   {
+     *     "language": "nl",
+     *     "filename": "**​/dutch/**​/.txt"
```

Comment:
I think you meant: `"filename": "**​/dutch/**​/*.txt"`

Reply:
yes.

can you offer this as a suggestion so i can click the button "commit suggestion"?